zap/src/main/resources/org/zaproxy/zap/resources/xml/report.md.xsl

```diff
@@ -24,6 +24,17 @@
 | Low | <xsl:value-of select="count(descendant::alertitem[riskcode='1'])"/> |
 | Informational | <xsl:value-of select="count(descendant::alertitem[riskcode='0'])"/> |
 
+## Alerts
+
+<xsl:for-each select="descendant::alertitem">
+  <xsl:sort order="descending" data-type="number" select="riskcode"/>
+  <xsl:sort order="descending" data-type="number" select="confidence"/>
+  <xsl:sort order="ascending" data-type="text" select="name"/>
+  <xsl:if test="position()=1">
+| Name | Number of Alerts |
```

Comment:
This can be added before the loop (if there are no rows it's rendered as an empty table).

Reply:
Okay, I'll do that.